src/rgw/rgw_gc.cc

```diff
@@ -314,10 +314,10 @@ class RGWGCIOManager {
 public:
   RGWGCIOManager(const DoutPrefixProvider* _dpp, CephContext *_cct, RGWGC *_gc) : dpp(_dpp),
                                                   cct(_cct),
-                                                  gc(_gc),
-                                                  remove_tags(cct->_conf->rgw_gc_max_objs),
-                                                  tag_io_size(cct->_conf->rgw_gc_max_objs) {
+                                                  gc(_gc) {
     max_aio = cct->_conf->rgw_gc_max_concurrent_io;
```

Comment:
this slightly scrambles indentation, maybe try to align all parts of the ctor-initializer list uniformly?

Reply:
sure, done